hibernate-core/src/main/java/org/hibernate/boot/internal/MetadataBuilderImpl.java

```diff
@@ -951,6 +958,11 @@ public boolean isXmlMappingEnabled() {
 			return xmlMappingEnabled;
 		}
 
+		@Override
+		public boolean disallowExtensionsInCdi() {
```

Comment:
Hmm, this looks strange to me
`disallowExtensionsInCdi()` returns `allowExtensionsInCdi`. Shouldn't it return `!allowExtensionsInCdi`?